### ISSUE TITLE
1.0.0-release

### ISSUE THREAD

**Issue by iwburns** [iwburns/id-tree#13]
Wanted to create this so we can group changes together for the 1.0.0-release that I hope to get going here soon.

Related PRs:
- #12 (now merged into branch 1.0.0-release)
- #19

**Comment by Drakulix** [comment]
I would like to see #11 fixed before 1.0.0.
This might result in a breaking change for `move_node_to_parent`.

Swap ( #6 ) on the other side should be able to be resolved fairly easy after #11 is done and should not result in a breaking change.

**Comment by Drakulix** [comment]
Is #16 now the last open bug to solve before solving this?

**Comment by iwburns** [comment]
I believe so yes (except that #16 is now #18 if we decide to go that route instead) .

**Comment by iwburns** [comment]
@Drakulix 1.0.0 is up on crates.io, just FYI

**Comment by Drakulix** [comment]
That was quicker than I expected! Thanks!
